Prettier check failing on CI for src/api/status/src/server.js

```
> prettier --check "./**/*.{md,jsx,json,html,css,js,yml}"

Checking formatting...
[warn] src/api/status/src/server.js
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @senecacdot/telescope@2.3.0 prettier-check: `prettier --check "./**/*.{md,jsx,json,html,css,js,yml}"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @senecacdot/telescope@2.3.0 prettier-check script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/src/api/status/src/server.js
+++ b/src/api/status/src/server.js
@@ -15,7 +15,11 @@ const satelliteOptions = {
           'fonts.googleapis.com',
           'cdn.jsdelivr.net',
         ],
-        connectSrc: ["'self'", '*.fontawesome.com', `${process.env.API_HOST.replace(/(^\w+:|^)\/\//, '')}:4000`],
+        connectSrc: [
+          "'self'",
+          '*.fontawesome.com',
+          `${process.env.API_HOST.replace(/(^\w+:|^)\/\//, '')}:4000`,
+        ],
         fontSrc: ["'self'", 'data:', 'https:', '*.fontawesome.com'],
         imgSrc: ["'self'", 'data:', 'https:'],
       },

